vscode/README.md

```diff
@@ -331,3 +331,16 @@ When `rubyLsp.formatter` is set to `auto`, Ruby LSP tries to determine which for
 
 If the bundle has a **direct** dependency on a supported formatter, such as `rubocop` or `syntax_tree`, that will be used.
 Otherwise, formatting will be disabled and you will need add one to the bundle.
+
+Here's a code example:
+
+```ruby
+# Gemfile
+
+group :development, :test do
+  gem "rubocop"
+end
+```
+
+Make sure to run `bundle install` after making the above change.
+
```

Comment:
Is this the final new line or an extra empty line at the end? If it's an empty line, can we please remove it?